files/ru/web/api/html_drag_and_drop_api/drag_operations/index.md

```diff
@@ -231,24 +231,24 @@ However, you can also update the user interface with an insertion point or highl
 
-The `{{event("dragover")}}` event will fire at the element the mouse is pointing at. Naturally, you may need to move the insertion marker around a `{{event("dragover")}}` event as well. You can use the event's {{domxref("MouseEvent.clientX","clientX")}} and {{domxref("MouseEvent.clientY","clientY")}} properties as with other mouse events to determine the location of the mouse pointer.
+The `[`dragover`](/ru/docs/Web/Events/dragover)` event will fire at the element the mouse is pointing at. Naturally, you may need to move the insertion marker around a `[`dragover`](/ru/docs/Web/Events/dragover)` event as well. You can use the event's {{domxref("MouseEvent.clientX","clientX")}} and {{domxref("MouseEvent.clientY","clientY")}} properties as with other mouse events to determine the location of the mouse pointer.
 
-Finally, the `{{event("dragleave")}}` event will fire at an element when the drag leaves the element. This is the time when you should remove any insertion markers or highlighting. You do not need to cancel this event. Any highlighting or other visual effects specified using the `:-moz-drag-over` pseudoclass will be removed automatically. The `{{event("dragleave")}}` event will always fire, even if the drag is cancelled, so you can always ensure that any insertion point cleanup can be done during this event.
+Finally, the `[`dragleave`](/ru/docs/Web/Events/dragleave)` event will fire at an element when the drag leaves the element. This is the time when you should remove any insertion markers or highlighting. You do not need to cancel this event. Any highlighting or other visual effects specified using the `:-moz-drag-over` pseudoclass will be removed automatically. The `[`dragleave`](/ru/docs/Web/Events/dragleave)` event will always fire, even if the drag is cancelled, so you can always ensure that any insertion point cleanup can be done during this event.
 
 ## Performing a Drop
 
 When the user releases the mouse, the drag and drop operation ends.
 
-If the mouse is released over an element that is a valid drop target, that is, one that cancelled the last `{{event("dragenter")}}` or `{{event("dragover")}}` event, then the drop will be successful, and a `{{event("drop")}}` event will fire at the target. Otherwise, the drag operation is cancelled, and no `{{event("drop")}}` event is fired.
+If the mouse is released over an element that is a valid drop target, that is, one that cancelled the last `[`dragenter`](/ru/docs/Web/Events/dragenter)` or `[`dragover`](/ru/docs/Web/Events/dragover)` event, then the drop will be successful, and a `[`drop`](/ru/docs/Web/Events/drop)` event will fire at the target. Otherwise, the drag operation is cancelled, and no `[`drop`](/ru/docs/Web/Events/drop)` event is fired.
```

Comment:
```suggestion
If the mouse is released over an element that is a valid drop target, that is, one that cancelled the last [`dragenter`](/ru/docs/Web/Events/dragenter) or [`dragover`](/ru/docs/Web/Events/dragover) event, then the drop will be successful, and a [`drop`](/ru/docs/Web/Events/drop) event will fire at the target. Otherwise, the drag operation is cancelled, and no [`drop`](/ru/docs/Web/Events/drop) event is fired.
```